docs/victoriametrics-cloud/README.md

```diff
@@ -0,0 +1,14 @@
+* [Overview of VictoriaMetrics Cloud](./overview.md)
+* [Quick Start](./quickstart.md)
+
+## Guides
+* [Understand Your Setup Size](../guides/understand-your-setup-size.md)
+* [Alerting & recording rules with Alertmanager configuration for VictoriaMetrics Cloud deployment](./alertmanager-setup-for-deployment.md)
+* [Kubernetes Monitoring with VictoriaMetrics Cloud](./how-to-monitor-k8s.md)
+* [Setup Notifications](./setup-notifications.md)
+* [User Management](./user-managment.md)
+
+Learn more about VictoriaMetrics Cloud:
+* [VictoriaMetrics Cloud announcement](https://victoriametrics.com/blog/victoriametrics-cloud-announcement)
```

Comment:
The link returns 404. It should be `https://victoriametrics.com/blog/managed-victoriametrics-announcement`

Reply:
As far as i can see it should be https://victoriametrics.com/blog/introduction-to-managed-monitoring/